.github/workflows/ci.yml

```diff
@@ -30,4 +30,5 @@ jobs:
       - run: ./setup.sh
       - run: rustup component add rustfmt --toolchain nightly
       - run: rustup component add clippy --toolchain nightly
+      - run: cargo +stable install cargo-audit
```

Comment:
Why `+stable`?

Reply:
We always install with +stable , i.e., see:
- `cargo +stable install cargo-fuzz` in `./fuzzing_setup.sh`
- `cargo +stable install elf2tab --version 0.10.2 --root elf2tab/` in `setup.sh`
```

My personal experience is that sometimes tools don't compile on the nightly versions we are running. But they should always run with stable?

Reply:
I'm not sure there's any guarantee a crate compiles on either stable or nightly (it happens that people just do breaking change and something stops compiling on stable). It seems simpler to always use the same toolchain everywhere. Also it's a bit ugly to force-install tools globally for the user, but that would be more work to fix.

No strong opinion, just curious why using `+stable`.

Reply:
In that case, I'll go with `stable` as the status quo that worked for me.